### PR TITLE
feat: show repl_time as seconds per MB

### PR DESCRIPTION
**Pull request by olizilla**
using microseconds gives us numbers in a range that is much easier
to reason about as human.

screenshot 2019-03-01 at 10 50 32

License: MIT
Signed-off-by: Oli Evans <oli@tableflip.io>

### CODE REVIEW

**Comment by olizilla** [comment]
@dignifiedquire here it is in seconds per kb flavour

screenshot 2019-03-01 at 11 17 58

Which one you like?

**Comment by dignifiedquire** [comment]
given these numbers are still pretty small maybe `seconds/MB`? that would at least for the current values gives us niceish number

**Comment by olizilla** [comment]
Now in seconds/Mb flavour
screenshot 2019-03-01 at 11 58 55

**Comment by dignifiedquire** [comment]
love it

File: src/App.js [hunk 1]
```diff
@@ -90,8 +89,8 @@ class App extends Component {
     }
 
     const data = leaderboardData
-      .map(d => ({ ...d, perByteTime: (d.repl_time * 1000) / d.params.size }))
-      .sort((a, b) => a.perByteTime - b.perByteTime)
+      .map(d => ({ ...d, secondsPerMBTime: d.repl_time / (d.params.size / 1e+6) }))
```
Comment (dignifiedquire) [inline src/App.js]: please use `1024 * 1024`

Comment (dignifiedquire) [reply src/App.js]: so it is `MiB` as that is the unit we use in other places

Comment (dignifiedquire) [reply src/App.js]: sorry for being imprecise before

**Comment by alanshaw** [comment]
Resolved by https://github.com/filecoin-project/replication-game-leaderboard/pull/15

**Pull request closed without merging**